connectors/activemq/src/main/java/forklift/connectors/ActiveMQConnector.java

```diff
@@ -100,6 +109,31 @@ public ForkliftConsumerI getTopic(String name)
     }
 
     @Override
+    public ForkliftConsumerI getConsumerForSource(SourceI source) throws ConnectorException {
+        return source
+            .apply(QueueSource.class, queue -> getQueue(queue.getName()))
+            .apply(TopicSource.class, topic -> getTopic(topic.getName()))
+            .apply(GroupedTopicSource.class, topic -> getGroupedTopic(topic))
+            .apply(RoleInputSource.class, roleSource -> {
+                final ForkliftConsumerI rawConsumer = getConsumerForSource(roleSource.getActionSource(this));
+                return new RoleInputConsumerWrapper(rawConsumer);
+            })
+            .elseUnsupportedError();
+    }
+
+    private ForkliftConsumerI getGroupedTopic(GroupedTopicSource source) throws ConnectorException {
+        if (!source.groupSpecified())
+            throw new ConnectorException("No consumer group specified");
+
+        // Sanitize the group and topic names so that group and topic names don't accidentally
+        // mess up ActiveMQ's pattern matching
+        final String groupName = source.getGroup().replaceAll("\\.", "_");
+        final String topicName = source.getName().replaceAll("\\.", "_");
+
+        return getConsumerForSource(new QueueSource("Consumer." + groupName + ".VirtualTopic." + topicName));
```

Comment:
The idea here is to map `GroupedTopicSource`s to a virtual topic in AMQ.
Since `GroupedTopic`s aren't a first class source in AMQ (at least AFAIK), we have to use a queue with the proper naming convention.

Also if I mapped a `GroupedTopicSource` to a `GroupedTopicSource`, wouldn't that cause infinite recursion?